### PR TITLE
Fix how to use ipvs mode by kubeadm

### PR DESCRIPTION
**Pull request by riverzhang**
**What this PR does / why we need it**:
how to use ipvs mode by kubeadm
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by neolit123** [comment]
@riverzhang 

how about removing any example kubeadm configs from this document and only linking externally to the `kubeadm-init` page?
the config is going to change multiple times in the near future and this is yet another document that has to be maintained.

@kubernetes/sig-cluster-lifecycle-pr-reviews 
/area kubeadm
/kind documentation
/hold

**Comment by riverzhang** [comment]
@neolit123  Thanks
add to kubeadm-init doc https://github.com/kubernetes/website/pull/10162

**Comment by neolit123** [comment]
> @neolit123 Thanks
> add to kubeadm-init doc kubernetes/website#10162

@riverzhang i meant something in the lines of:
- we shouldn't have:
```
kind: yyyy
apiVersion: xxxx
```
for any of the kubeadm examples.

you can just provide:
```
kubeProxy:
  config:
    featureGates:
      SupportIPVSProxyMode: true
    mode: ipvs
```
and say that after 1.11 `SupportIPVSProxyMode` is true by default.

the fully config is outlined already here:
https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file

but if you want to maintain the README.md with these changes it's OK, but please mind that the config is going to change mutliple times in the next releases.

File: pkg/proxy/ipvs/README.md [hunk 1]
```diff
@@ -338,10 +338,23 @@ Kube-proxy will run in iptables mode by default in a cluster deployed by [kubead
...
+    featureGates:
+      SupportIPVSProxyMode: true
+    mode: ipvs
+...
+```
+2. Kubernetes v1.11 and later
+```json
+kind: MasterConfiguration
+apiVersion: kubeadm.k8s.io/v1alpha2
+...
```
Comment (neolit123) [inline pkg/proxy/ipvs/README.md]: just remove both kubeadm config examples in this document and do this instead:

----------


If you are using kubeadm with a [configuration file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file), you have to add `mode: ipvs` and also add `SupportIPVSProxyMode: true` below the `kubeProxy` field as part of the kubeadm configuration.
```
kubeProxy:
  config:
    featureGates:
      SupportIPVSProxyMode: true
    mode: ipvs
```
Note that in Kubernetes 1.11 and later, `SupportIPVSProxyMode` is set to `true` by default.

----------

Comment (riverzhang) [reply pkg/proxy/ipvs/README.md]: @neolit123   Thanks a lot.

File: pkg/proxy/ipvs/README.md [hunk 2]
```diff
@@ -334,19 +334,19 @@ export KUBE_PROXY_MODE=ipvs
 
 ### Cluster Created by Kubeadm
 
-Kube-proxy will run in iptables mode by default in a cluster deployed by [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/).
-
-If you are using kubeadm with a [configuration file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file), you can specify the ipvs mode adding `SupportIPVSProxyMode: true` below the `kubeProxy` field.
+If you are using kubeadm with a [configuration file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file), you have to add mode: ipvs and also add SupportIPVSProxyMode: true below the kubeProxy field as part of the kubeadm configuration.
```
Comment (neolit123) [inline pkg/proxy/ipvs/README.md]: add code quotes around `mode: ipvs`, `SupportIPVSProxyMode: true` and `kubeProxy`?

File: pkg/proxy/ipvs/README.md [hunk 3]
```diff
@@ -334,19 +334,19 @@ export KUBE_PROXY_MODE=ipvs
...
-apiVersion: kubeadm.k8s.io/v1alpha1
 ...
 kubeProxy:
   config:
+    featureGates:
+      SupportIPVSProxyMode: true
     mode: ipvs
 ...
 ```
+Note that in Kubernetes 1.11 and later, SupportIPVSProxyMode is set to true by default.
```
Comment (neolit123) [inline pkg/proxy/ipvs/README.md]: `SupportIPVSProxyMode` is set to `true`

File: pkg/proxy/ipvs/README.md (hunk 2 above)
Comment (riverzhang) [reply pkg/proxy/ipvs/README.md]: sorry, fix it

File: pkg/proxy/ipvs/README.md (hunk 3 above)
Comment (riverzhang) [reply pkg/proxy/ipvs/README.md]: sorry fix it.

**Comment by timothysc** [comment]
milestone applied.

**Pull request merged**